README.md

```diff
@@ -36,19 +53,18 @@ git_repository(
     commit = "{HEAD}",
```

Comment:
I know this already exists but it seems like worrying advice, since people will just copy and paste this.

Reply:
I think that's ok at the moment. I'd rather they copy/paste, get immediately broken, and look for a recent commit, than that they copy/paste, don't get broken right away, but get an old commit (assuming we're not updating this readme every commit).

In the near future I'll cut a tag `0.1.0`, and change this to a true copy/paste recipe. (Tony is also working on a way to auto-generate recipes as part of the tag description, which sounds pretty cool.)

Reply:
SGTM